installer/config.yml

```diff
@@ -85,3 +85,170 @@ playout:
+        mount: main
+        # Icecast source user.
+        # > default is source
+        source_user: source
+        # Icecast source password.
+        # > this field is REQUIRED
+        source_password: hackme
+        # Icecast admin user.
+        # > default is admin
+        admin_user: admin
+        # Icecast admin password. If not defined, stats will not be collected.
+        admin_password: hackme
+
+        # Icecast output audio.
+        audio:
+          # Icecast output audio format.
+          # > must be one of (aac, mp3, ogg, opus)
+          # > this field is REQUIRED
+          format: ogg
+          # Icecast output audio bitrate.
+          # > must be one of (32, 48, 64, 96, 128, 160, 192, 224, 256, 320)
+          # > this field is REQUIRED
+          bitrate: 256
+
+          # format=ogg only field: Embed metadata (track title, artist, and show name)
+          # in the output stream. Some bugged players will disconnect from the stream
+          # after every songs when playing ogg streams that have metadata information
+          # enabled.
+          # > default is false
+          enable_metadata: false
+
+        # Icecast stream name.
+        name: LibreTime!
+        # Icecast stream description.
+        description: LibreTime Radio!
+        # Icecast stream website.
+        website: https://libretime.org
+        # Icecast stream genre.
+        genre: various
+
+    # Shoutcast output streams.
+    # > max items is 1
+    shoutcast:
+      - # Whether the output is enabled.
+        # > default is false
+        enabled: false
+        # Output public url. If not defined, the value will be generated from
+        # the [general.public_url] hostname and the output port.
+        url:
+        # Shoutcast server host.
+        # > default is localhost
+        host: localhost
+        # Shoutcast server port.
+        # > default is 8000
+        port: 8000
+        # Shoutcast source user.
+        # > default is source
+        source_user: source
+        # Shoutcast source password.
+        # > this field is REQUIRED
+        source_password: hackme
+        # Shoutcast admin user.
+        # > default is admin
+        admin_user: admin
+        # Shoutcast admin password. If not defined, stats will not be collected.
+        admin_password: hackme
+
+        # Shoutcast output audio.
+        audio:
+          # Shoutcast output audio format.
+          # > must be one of (aac, mp3)
+          # > this field is REQUIRED
+          format: mp3
+          # Shoutcast output audio bitrate.
+          # > must be one of (32, 48, 64, 96, 128, 160, 192, 224, 256, 320)
+          # > this field is REQUIRED
+          bitrate: 256
+
+        # Shoutcast stream name.
+        name: LibreTime!
+        # Shoutcast stream website.
+        website: https://libretime.org
+        # Shoutcast stream genre.
+        genre: various
+
+    # Line outputs.
+    # > max items is 1
+    line:
```

Comment:
Is line outputs a good name ? Should we consider hardware output ? Unsure which name is the best to group all the system outputs.

Reply:
Maybe `device` or `system`. We have to group the following alsa, ao, oss, portaudio, pulseaudio.

This list could probably be extended at some point, with gstreamer, and other outputs supported by liquidsoap.

Reply:
I like `system` , think it describes what it is better than device